.swiftlint.yml

```diff
@@ -1,2 +1,5 @@
 disabled_rules: # rule identifiers to exclude from running
   - large_tuple
+  
```

Comment:
@tjprescott @sacheu making a new addition to swiftlint line length rule to ignore comments.